stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -69,7 +143,185 @@ private static String getIdAttribute(View element) {
+        }
+      }
+
+      if (!caught) {
+        convertValueToCSSString(
+            element,
+            property.getName(),
+            value,
+            property.getAnnotation(),
+            styles);
+      }
+    }
+  }
+
+  private static boolean canIntBeMappedToString(@Nullable ViewDebug.ExportedProperty annotation) {
+    return annotation != null && annotation.mapping() != null && annotation.mapping().length > 0;
+  }
+
+  private static String mapIntToStringUsingAnnotation(
+      int value,
+      @Nullable ViewDebug.ExportedProperty annotation) {
+    if (!canIntBeMappedToString(annotation)) {
+      throw new IllegalStateException("Cannot map using this annotation");
+    }
+
+    for (ViewDebug.IntToString map : annotation.mapping()) {
+      if (map.from() == value) {
+        return map.to();
+      }
+    }
+
+    // no mapping was found even though one was expected ):
+    return NONE_MAPPING;
+  }
+
+  private static boolean canFlagsBeMappedToString(@Nullable ViewDebug.ExportedProperty annotation) {
+    return annotation != null && annotation.flagMapping() != null && annotation.flagMapping().length > 0;
+  }
+
+  private static String mapFlagsToStringUsingAnnotation(
+      int value,
+      @Nullable ViewDebug.ExportedProperty annotation) {
+
+    if (!canFlagsBeMappedToString(annotation)) {
+      throw new IllegalStateException("Cannot map using this annotation");
+    }
+
+    StringBuilder stringBuilder = new StringBuilder();
+    boolean atLeastOneFlag = false;
+
+    for (ViewDebug.FlagToString flagToString : annotation.flagMapping()) {
+      if (flagToString.outputIf() == ((value & flagToString.mask()) == flagToString.equals())) {
+        if (atLeastOneFlag) {
+          stringBuilder.append(" | ");
+        }
+
+        stringBuilder.append(flagToString.name());
+        atLeastOneFlag = true;
+      }
+    }
+
+    if (atLeastOneFlag) {
+      return stringBuilder.toString();
+    } else {
+      return NONE_MAPPING;
+    }
+  }
+
+  private static boolean isDefaultValue(Number value, @Nullable ViewDebug.ExportedProperty annotation) {
+    if (annotation != null) {
+      if (annotation.mapping().length > 0) {
+        return false;
+      }
+    }
+
+    return value.floatValue() == 0.0f;
+  }
+
+  private String convertGetterNameToCssName(String getterName) {
+    // Split string by uppercase characters. Thankfully since
+    // this is the android source we don't have to worry about
+    // internationalization funk.
+
+    String[] words = getWordBoundaryPattern().split(getterName);
+
+    StringBuilder result = new StringBuilder();
+
+    for (int i = 0; i < words.length; i++) {
+      if (words[i].equals("get") || words[i].equals("m")) {
+        continue;
+      }
+
+      result.append(words[i].toLowerCase());
+
+      if (i < words.length - 1) {
+        result.append('-');
+      }
+    }
+
+    return result.toString();
+  }
+
+  private void convertValueToCSSString(
```

Comment:
Since this method is doing the `StyleAccumulator.store()` call, it's not really converting a value to a "CSS string". Also, the pattern for methods that take an `Accumulator` is to use "get" for the verb. So maybe we just call this method `getStyleFromPropertyValue`.